### PR TITLE
missing files

### PR DESCRIPTION
**Pull request by cedrict**
Something went wrong when this benchmark was merged during the hackathon. The manual is up-to-date w.r.t. to this benchmark but the .cc and CMakeList files are still missing.

### CODE REVIEW

File: benchmarks/viscosity_grooves/viscosity_grooves.cc [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+      double
+      ViscosityGrooves_pressure (const Point<2> &pos,
+                                 const GeometryModel::Interface<2> &geometry_model)
+      {
+        const double x = pos[0];
+        const double y = pos[1];
+        const GeometryModel::Box<2> *geometry
+          = dynamic_cast<const GeometryModel::Box<2>*> (&geometry_model);
+        const double L=geometry->get_extents()[0];
+        return x*x*y*y+x*y+5 - pow(L,4)/9-pow(L,2)/4-5;
```
Comment (tjhei) [inline benchmarks/viscosity_grooves/viscosity_grooves.cc]:
```suggestion
        return x*x*y*y+x*y+5. - std::pow(L, 4.)/9.-std::pow(L,2.)/4.-5.;
```

File: benchmarks/viscosity_grooves/viscosity_grooves.cc [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+                                         &comp_u);
+      VectorTools::integrate_difference (this->get_mapping(),this->get_dof_handler(),
+                                         this->get_solution(),
+                                         *ref_func,
+                                         cellwise_errors_pl2,
+                                         quadrature_formula,
+                                         VectorTools::L2_norm,
+                                         &comp_p);
+
+      const double u_l1 =  Utilities::MPI::sum(cellwise_errors_u.l1_norm(),this->get_mpi_communicator());
```
Comment (tjhei) [inline benchmarks/viscosity_grooves/viscosity_grooves.cc]: can you please switch this to ``compute_global_error`` like I did in all other benchmarks in the last few weeks?

**Comment by tjhei** [comment]
@cedrict ?

File: benchmarks/viscosity_grooves/viscosity_grooves.cc (hunk 2 above)
Comment (cedrict) [reply benchmarks/viscosity_grooves/viscosity_grooves.cc:451]: this seems to have disappeared from what is now merged in the code

**Comment by cedrict** [comment]
This is superseded by  #3568 . closing it

**Pull request closed without merging**